docusaurus.config.js

```diff
@@ -412,6 +412,7 @@ module.exports = {
         path: "kb",
         blogTitle: "Temporal Platform knowledge base",
         blogSidebarTitle: "Recent KB articles",
```

Comment:
If we change `blogSidebarCount` to `ALL` instead of a higher number, should we change `blogSidebarTitle` to simply "KB articles"?

Reply:
Nice catch.